config/locales/mailers/en.yml

```diff
@@ -50,6 +50,25 @@ en:
         part1: "Your work \"%{title}\" was deleted from the Archive by a site admin."
         part2: "If your work was part of an import project managed by our Open Doors team, please contact Open Doors (%{opendoors_link}) with any further questions."
         part3: "If it's possible your work violated the Archive's Terms of Service, please contact our Abuse Committee (%{abuse_link})."
+    admin_hidden_work_notification:
+      subject: "[%{app_name}] Your work has been hidden by the Abuse Team"
+      hello: "Dear %{name},"
+      tos: "Terms of Service"
+      abuse_committee: "contact Policy & Abuse"
+      sincerely: "Sincerely,"
+      sender: "AO3 Abuse"
+      html:
+        part1: "Your work %{title} has been hidden by the Policy & Abuse team and is no longer publicly accessible."
+        part2: "While your work is hidden, you will still be able to access it through the link provided above, but it will not be listed on your works page, and it won't be available to other users of the Archive."
+        part3: "Please check your email, including your spam folder, as the Policy & Abuse team may have already contacted you explaining why your work was hidden."
+        part4: "If your work was hidden due to being in violation of the Archive of Our Own's %{tos_link}, you will be required to take action to correct the violation. Failure to bring your work into compliance with the Terms of Service may lead to your work being deleted from the Archive."
+        part5: "If you are uncertain why your work was hidden, and you have not received further communication regarding this matter, please %{abuse_link} directly."
```

Comment:
Could you change the key names per [the guidelines](https://github.com/otwcode/otwarchive/wiki/Internationalization-(i18n)-Standards)?

>**Keys should be descriptive and not be numbered sequentially.** Avoid key names like part1 or para1. If you have a five-paragraph email, with each paragraph called para1, para2, and so on, when you insert a new paragraph between first and second paragraphs, it will require the keys for existing paragraphs to be renumbered. This will require our Translation team to reenter the text for all of those paragraphs in every language.

Going in order, they could probably be renamed:
* hidden
* access
* check_email
* tos_violation
* help

It also looks like part2/access and part3/check_email are the same in both the HTML and text versions, so you only need to include it once, directly under `admin_hidden_work_notification` instead of under both `html` and `text`.